setup_webui.py

```diff
@@ -34,7 +34,7 @@
     from setuputil import get_rucio_version
 
 name = 'rucio-webui'
-packages = ['rucio', 'rucio.web', 'rucio.web.ui', 'rucio.web.ui.common']
+packages = ['rucio', 'rucio.web', 'rucio.web.ui']
```

Comment:
`packages = ['rucio', 'rucio.web', 'rucio.web.ui', 'rucio.web.ui.flask', 'rucio.web.flask.common']` ?

Reply:
Yes, you are right. Not sure why we didn't add this in the first place. Will add.

Reply:
probably because this setup.py is long forgotten :D

Reply:
Actually, `rucio.web.flask.common ` doesn't exist :-)

Reply:
I missed the `ui`, correct. Should've been `rucio.web.ui.flask.common`